### ISSUE TITLE
Add support for docker images with long git rev strings

### ISSUE THREAD

**Issue by tlianza** [cloudflare/cfdeploy#8]
At present cfdeploy runs git rev-parse --short which doesn't have a guaranteed /predictable behavior across repos or git versions. In order to integrate cfdeploy with other tools, it would be great to support either a custom length, or full length git revision as a variable.

**Comment by ryan0x44** [comment]
Agreed! I suggest we add `GitRev` (or if that's ambiguous, `GitRevFull`), and then if you want to truncate you can just do something like `tagTemplate: "{{ printf "%.10s" .GitRev }}"` to truncate to 10 characters. Thoughts?
